src/Model/ValueObject/MediaObject/ImagesToMediaObjectReferencesConvertor.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Model\ValueObject\MediaObject;
+
+use CultuurNet\UDB3\Media\MediaObject;
+use CultuurNet\UDB3\Media\MediaObjectRepository;
+use CultuurNet\UDB3\Model\ValueObject\MediaObject\MediaObject as MediaObjectDto;
+
+class ImagesToMediaObjectReferencesConvertor
```

Comment:
As discussed through Slack a better approach could be to work with `Image` inside the `ImageEditor` and use the `ImageNormalizer` to get the URL of the thumbnail. But this can be done in a follow-up pull request.